README.md

```diff
@@ -1 +1,110 @@
-# nv2
\ No newline at end of file
+# Notary V2 (nv2) - Prototype
+
+nv2 is an incubation and prototype for the [Notary v2][notary-v2] efforts, securing artifacts stored in [distribution-spec][distribution-spec] based registries.
+The `nv2` prototype covers the scenarios outlined in [notaryproject/requirements](https://github.com/notaryproject/requirements/blob/master/scenarios.md#scenarios). It also follows the [prototyping approach described here](https://github.com/stevelasker/nv2#prototyping-approach).
+
+![nv2-components](media/notary-e2e-scenarios.png)
+
+To enable the above workflow:
+
+- The nv2 client (1) will sign any OCI artifact type (2) (including a Docker Image, Helm Chart, OPA, SBoM or any OCI Artifact type), generating a Notary v2 signature (3)
+- The [ORAS][oras] client (4) can then push the artifact (2) and the Notary v2 signature (3) to an OCI Artifacts supported registry (5)
+- In a subsequent prototype, signatures may be retrieved from the OCI Artifacts supported registry (5)
+
+![nv2-components](media/nv2-client-components.png)
+
+## Table of Contents
+
+1. [Scenarios](#scenarios)
+1. [nv2 signature spec](./docs/signature/README.md)
+1. [nv2 signing and verification docs](docs/nv2/README.md)
+1. [OCI Artifact schema for storing signatures](docs/artifact/README.md)
+1. [nv2 prototype scope](#prototype-scope)
+
+## Scenarios
+
+The current implementation focuses on x509 cert based signatures. Using this approach, the digest and references block are signed, with the cert Common Name required to match the registry references. This enables both the public registry and private registry scenarios.
```

Comment:
An image can be renamed, moved to a different registry, different tag, etc. 
What this block shows is: what was it when it was signed.
When you run example.com/hello-world:v1 from myregistry.io/hello-you:v1-blah, the validation would pass as long as the digest hasn't changed.
However, you can also add another registry.corp.io signature for: registry.corp.io/hello-you:v1-blah, and then limit deployments to being signed by registry.corp.io and pulled from registry.corp.io. The signature validation would come from Notary v2. The additional "policy" would be something incorporated into a policy managmenet solution like OPA or something else. 
Think of it as an extra layer of security, we may like. Or, we may not. See comment above: https://github.com/notaryproject/nv2/pull/2#discussion_r461807960

Reply:
Roughly: the signature attests that the signer (Foo) called the image by this name (foo.example.com/myimage:latest), and a validator can implement a policy such that it only trusts the name (foo.example.com/myimage:*) if it was signed by this key (Foo)?

Reply:
Yup, But, I suspect companies would more likely re-sign for themselves and trust their own registry. The main point is an x509 signature can only sign artifacts where the CN matches the registry name.
As we're discussing the larger question of key types, this may become less possible. 
But, is it interesting?

Reply:
@SteveLasker 
I think we need clarification regarding the "re-sign" scenario you mentioned.
More specifically, should the signature by `registry.corp.io`  be independent of the signature by `example.com`?
Is it reasonable that when the signature by `example.com` is revoked, the signature by `registry.corp.io` should be invalid immediately?

Reply:
Hi Daniel,
Great question. Today, the signatures are independent. The reason a signature may get revoked may or may not have anything to do with subsequent signatures. But, the information is super important. I'd suggest this is more of a policy managmenet decision that makes use of the information, rather than a hard rule the spec would enforce. 
To your point, once we've all gotten comfortable with the prototype, we should absolutely clarify this in the spec. 
I think the other question buried here is the continual question of policy vs. data separation. 
I'm suggesting signatures are data, enabling policy by external systems. 
We must support the ability to invalidate some data - such as the wabbit-networks signature may be revoked. If acme-rockets wants to keep or invalidate their signature is a policy decision. 
Thoughts?